style.css

```diff
@@ -188,26 +234,87 @@ html, body {
         width: 100%;
         height: 100%;
         padding: 70px 10px 20px;
-        background-color: #ffffff;
-        transition: 0.3s;
+        background-color: var(--main-bg-color);  
+        transition: 0.5s;
```

Comment:
do you describe property which have transition?

Reply:
Yes, this menu drop down when we click on the burger